ci/sge/hooks/post_build

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
```

Comment:
this is automatically executed once the master image is build

Reply:
This file seems related to autodocker IIUC (e.g. https://docs.docker.com/docker-hub/builds/advanced/#custom-build-phase-hooks)? Can we remove it?